core/src/Streamly/Internal/Data/Ring.hs

```diff
@@ -174,7 +175,6 @@ fromArray = undefined
 
 -- | Modify a given index of a ring array using a modifier function.
 --
--- /Unimplemented/
```

Comment:
Why remove this?

Reply:
Ah, I was going to implement it. Then decided against it.